source/common/router/scoped_rds.h

```diff
@@ -0,0 +1,203 @@
+#pragma once
+
+#include <string>
+
+#include "envoy/api/v2/srds.pb.h"
+#include "envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.pb.h"
+#include "envoy/config/subscription.h"
+#include "envoy/stats/scope.h"
+
+#include "common/common/logger.h"
```

Comment:
Suggest moving more of this file into the `.cc` and then we lose the `.h` dependency here on as many files.

Reply:
Done.